hw/ip/otbn/dv/rig/rig/model.py

```diff
@@ -183,10 +183,11 @@ class Model:
     following the instruction stream to this point.
 
     '''
-    def __init__(self, dmem_size: int, fuel: int) -> None:
+    def __init__(self, dmem_size: int, fuel: int, pqc: int) -> None:
```

Comment:
It seems self.pqc is used as a boolean? If so, declare it (and use it) as type bool.